### PR TITLE
Clpeterson/fix/sweepers

### PR DESCRIPTION
**Pull request by CLPeterson**

### CODE REVIEW

**Review by kenanmamedoff — APPROVED** [review]

File: brd-ios/breadwallet/src/ViewControllers/Import/ImportKeyViewController.swift [hunk 1]
```diff
@@ -226,8 +226,12 @@ class ImportKeyViewController: UIViewController, Subscriber {
     
     private func submit(sweeper: WalletSweeper, fee: TransferFeeBasis) {
         guard let transfer = sweeper.submit(estimatedFeeBasis: fee) else {
-            importingActivity.dismiss(animated: true)
-            return showErrorMessage(L10n.Alerts.sendFailure)
+//            importingActivity.dismiss(animated: true)
+//            return showErrorMessage(L10n.Alerts.sendFailure)
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/ViewControllers/Import/ImportKeyViewController.swift:230]: should this be removed?

File: brd-ios/breadwallet/src/ViewControllers/Import/ImportKeyViewController.swift [hunk 2]
```diff
@@ -226,8 +226,12 @@ class ImportKeyViewController: UIViewController, Subscriber {
...
+            self.importingActivity.dismiss(animated: true) {
+                self.showErrorMessage(L10n.Alerts.sendFailure)
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/ViewControllers/Import/ImportKeyViewController.swift:232]: does import always end up with a failure?

File: brd-ios/breadwallet/src/ViewControllers/Import/ImportKeyViewController.swift (hunk 1 above)
Comment (CLPeterson) [reply brd-ios/breadwallet/src/ViewControllers/Import/ImportKeyViewController.swift:230]: Yes, I can remove.

File: brd-ios/breadwallet/src/ViewControllers/Import/ImportKeyViewController.swift (hunk 2 above)
Comment (CLPeterson) [reply brd-ios/breadwallet/src/ViewControllers/Import/ImportKeyViewController.swift:232]: The import can succeed. It only failed for me when the value of the paperwallet was too low. The code change enables the error message to properly display if the import does fail.

**Review by kenanmamedoff — APPROVED** [review]

**Review by kenanmamedoff — APPROVED** [review]

**Pull request merged**